implementations/rust/vault/src/types.rs

```diff
@@ -119,20 +123,24 @@ try_from_int_impl!(SecretPersistenceType, u128);
 pub enum SecretPurposeType {
     /// Key exchange
     KeyAgreement,
+    /// Epilogue
+    Epilogue,
```

Comment:
Can we get a more detailed comment about what `Epilogue` is?
What is it used for?

Reply:
This is workaround for atecc, because during epilogue of key agreement it uses different approach about which secrets go to which slots.